core/templates/about_api.html

```diff
@@ -37,10 +37,11 @@ <h2>The API</h2>
 <h3>Jump to:</h3> 
 <ul class="anchor"> 
 <li><a href="#search">Search</a> the newspaper directory and digitized page contents using OpenSearch.</li> 
-<!-- <li><a href="#autosuggest">Auto Suggest</a> API for looking up newspaper
-titles</li> Could not find this section so am removing the link -kmd -->
+{# <li><a href="#autosuggest">Auto Suggest</a> API for looking up newspaper titles</li> #}
```

Comment:
I changed it because comments like this that are clearly for developer notes / discussion shouldn't be in HTML output. It has no use for users of the website. It was right next to markup I needed to edit otherwise I wouldn't have touched it at all.

I think I'll just remove it entirely though as there isn't really any value for future developers either. Just noting/tracking removals is the job of source control, not comments :smile: